src/modules/75-cf/pages/onboarding/views/SelectEnvironmentView.tsx

```diff
@@ -59,92 +67,110 @@ export const SelectEnvironmentView: React.FC<SelectEnvironmentViewProps> = props
 
   const serverSide = props.language.type === PlatformEntryType.SERVER
 
+  const onCloseEnvDialog = (): void => {
+    setDialogOpen(false)
+    setCreateEnvName('')
+  }
+
   return (
-    <Container margin={{ top: 'large' }}>
-      <Layout.Vertical spacing="xsmall">
-        <Text className={css.selectEnvironment}>{getString('cf.onboarding.selectYourEnvironment')}</Text>
-        <Container className={css.environmentContainer}>
+    <Layout.Vertical spacing="xsmall" margin={{ top: 'xlarge' }} className={css.grid}>
+      <Text
+        font={{ variation: FontVariation.BODY1 }}
+        color={Color.GREY_800}
+        margin={{ top: 'medium', bottom: 'medium' }}
+      >
+        {getString('cf.onboarding.selectOrCreateEnvironment')}
+      </Text>
+      <Text font={{ variation: FontVariation.BODY }} color={Color.GREY_600}>
+        {getString('cf.onboarding.environmentDescription')}
+      </Text>
+      <Container>
+        <Layout.Horizontal spacing="small">
+          {loading && <Icon name="spinner" size={16} color="blue500" />}
+          {!loading && (
+            <>
+              {!!environments?.length && (
+                <Container width={250}>
+                  <EnvironmentSelect />
+                </Container>
+              )}
+              {createEnvName && (
+                <EnvironmentDialog
+                  createEnvFromInput={dialogOpen}
+                  createEnvName={createEnvName}
+                  onCreate={() => {
+                    setEnvironmentCreated(true)
+                    refetch()
+                    props.setApiKey(undefined)
+                    onCloseEnvDialog()
+                  }}
+                  buttonProps={{
+                    intent: Intent.NONE,
+                    text: getString('cf.onboarding.createEnv'),
+                    style: { color: 'var(--blue-500)', ...(environments?.length ? { border: 'none' } : {}) }
+                  }}
+                  onCloseDialog={() => {
+                    onCloseEnvDialog()
+                  }}
```

Comment:
Not a problem, so feel free to ignore, but you can simplify a wee bit
```suggestion
                  onCloseDialog={onCloseEnvDialog}
```